capellambse/model/crosslayer/interaction.py

```diff
@@ -17,10 +20,125 @@
 )
 
 
+@c.xtype_handler(None)
+class Execution(c.GenericElement):
+    """An execution."""
+
+    start = c.AttrProxyAccessor(c.GenericElement, "start")
+    finish = c.AttrProxyAccessor(c.GenericElement, "finish")
+
+
+@c.xtype_handler(None)
+class StateFragment(Execution):
+    """A state fragment."""
+
+    function = c.AttrProxyAccessor(c.GenericElement, "relatedAbstractFunction")
+
+
+@c.xtype_handler(None)
+class CombinedFragment(Execution):
+    """A combined fragment."""
+
+    operator = xmltools.AttributeProperty(
+        "_element", "operator", optional=True
+    )
+    operands = c.AttrProxyAccessor(
+        c.GenericElement, "referencedOperands", aslist=c.ElementList
+    )
+
+
+@c.xtype_handler(None)
+class InstanceRole(c.GenericElement):
+    """An instance role."""
+
+    instance = c.AttrProxyAccessor[c.GenericElement](
+        None, "representedInstance"
+    )
+
+
+@c.xtype_handler(None)
+class SequenceMessage(c.GenericElement):
+    """A sequence message."""
+
+    source = c.AttrProxyAccessor(c.GenericElement, "sendingEnd")
+    target = c.AttrProxyAccessor(c.GenericElement, "receivingEnd")
+
+
+class Event(c.GenericElement):
+    """Abstract super class of all events in a Scenario."""
+
+
+class EventOperation(Event):
+    """Abstract super class for events about operations."""
+
+    operation = c.AttrProxyAccessor(c.GenericElement, "operation")
+
+
+@c.xtype_handler(None)
+class ExecutionEvent(Event):
+    pass
```

Comment:
All other classes have docstrings instead of the pass statement. Even though the docstring doesn't add much more information. This is inconsistent with the other classes.

Reply:
```suggestion
    """An execution event."""
```